tests/playwright/src/model/pages/pods-details-page.ts

```diff
@@ -61,7 +61,7 @@ export class PodDetailsPage extends BasePage {
   }
 
   async getState(): Promise<string> {
-    const currentState = await this.page.getByRole('status').getAttribute('title');
+    const currentState = await this.page.getByRole('status').nth(0).getAttribute('title');
```

Comment:
Maybe we should change the HTML code of the pod details page by defining new attributes for the parent area where the pod status and buttons are located, instead of using the nth() function?

Reply:
I'm not sure I understand why you needed to add the nth(0) param, is there another cell in that row with the role of `status`? The methods that are calling this element seem to be working fine.

Reply:
I encountered an error while testing on a Windows machine. There was another element with the role of status.

Reply:
![Screenshot from 2024-07-19 15-51-03](https://github.com/user-attachments/assets/15cf269f-844d-4eda-8237-e4ea8492689c)

Reply:
Which button is this "Tasks"?

Reply:
I suppose that it is a bug, there isn't any 'Tasks' button on the test screenshots.

Reply:
Details page needs to get some ARIA functionality - #5966